src/Var.cc

```diff
@@ -470,6 +470,18 @@ void add_type(ID* id, TypePtr t, std::unique_ptr<std::vector<AttrPtr>> attr)
 		id->SetAttrs(make_intrusive<Attributes>(std::move(*attr), tnew, false, false));
 	}
 
+static std::set<std::string, std::less<>> all_module_names;
```

Comment:
```suggestion
static std::set<std::string> all_module_names;
```

Should work without, no?

Reply:
Yep, works and I can remove it. [Sonarlint suggested to add](https://rules.sonarsource.com/cpp/RSPEC-6045/) std::less<> and my understanding is that it can void creating string objects when doing lookups using `const char *` or `string_view`.

It won't matter here and if it looks noisy unusual better to drop.